pages/accept-financial-contributions.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from '@apollo/react-hoc';
+
+import AcceptFinancialContributions from '../components/accept-financial-contributions/index.js';
+import ErrorPage from '../components/ErrorPage';
+import AuthenticatedPage from '../components/AuthenticatedPage';
+import { withUser } from '../components/UserProvider';
+import Loading from '../components/Loading';
+import Container from '../components/Container';
+
+import { getCollectivePageQuery } from '../components/collective-page/graphql/queries';
+
+class AcceptFinancialContributionsPage extends React.Component {
+  static async getInitialProps({ query }) {
+    return {
+      slug: query && query.slug,
+      query,
```

Comment:
I realized today that using `withRouter` and `this.props.router.query` is more efficient than trying to pass `query` from the Page down to the component needing it (here the Picker). I recommend to use this strategy here, `query` can be ultimately removed.

Reply:
Updated to use `withRouter` everywhere, and will remember that for the future